string void_request
---
bool application
